modules/core/src/main/java/org/apache/ignite/internal/util/nio/GridNioMetricsListener.java

```diff
@@ -17,10 +17,12 @@
 
 package org.apache.ignite.internal.util.nio;
 
+import java.util.EventListener;
+
 /**
  * Metrics listener for NIO communication.
  */
-public interface GridNioMetricsListener {
+public interface GridNioMetricsListener extends EventListener {
```

Comment:
Why is it in this patch?

Reply:
Because we have exclusion from toString() for EventListener's